File: sol/1957/1957.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        letter = None
        streak = 0
        res = ''
        for c in s:
            if c == letter and streak == 2:
                continue
            elif c == letter:
                streak += 1
            else:
                letter = c
                streak = 1
            res += c
        return res

File: sol/1957/1957.cpp
struct Solution {
    string makeFancyString(string &s) {
        string res;
        char letter = ' ';
        int streak = 0;
        for (char c : s){
            if (c == letter && streak == 2){
                continue;
            } else if (c == letter){
                ++streak;
            } else {
                letter = c;
                streak = 1;
            }
            res += c;
        }
        return res;
    }
};
